.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,123 @@
+name: New issue for bug report
+description: Got a glitch, crash, or some other problem? If you're sure your issue is reproducible, feel free to report the issue to Natron's development team.
+title: "(Bug): <title>"
+labels: "type:bug"
+body:
+  # Greeter message
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting a bug! Please first check the boxes below to ensure that your bug report meets our requirements.
+  # Requirements checks
+  - type: checkboxes
+    attributes:
+      options:
+        - label: I'm using the latest version of Natron (not required but recommended)
+        - label: I've restarted Natron and the issue persists
+          required: true
+        - label: I've run Natron via the [command line](https://natron.readthedocs.io/en/rb-2.4/devel/natronexecution.html) and the issue persists
```

Comment:
Links that will break :(

Reply:
Yeah, same deal as before - unless we build a url redirector into the website itself this isn't going to get solved, but I think it's bearable for now.